--- a/OpenSim/Services/InventoryService/XInventoryService.cs	2026-01-04 07:12:30.372415465 +0000
+++ b/OpenSim/Services/InventoryService/XInventoryService.cs	2026-01-04 07:12:30.360414498 +0000
@@ -153,6 +153,12 @@
                 CreateFolder(principalID, rootFolder.ID, (int)FolderType.Trash, "Trash");
             if (!Array.Exists(sysFolders, delegate (XInventoryFolder f) { if (f.type == (int)FolderType.Settings) return true; return false; }))
                 CreateFolder(principalID, rootFolder.ID, (int)FolderType.Settings, "Settings");
+            if (!Array.Exists(sysFolders, delegate (XInventoryFolder f) { if (f.type == 48) return true; return false; }))
+                CreateFolder(principalID, rootFolder.ID, 48, "My Outfits");
+            if (!Array.Exists(sysFolders, delegate (XInventoryFolder f) { if (f.type == 50) return true; return false; }))
+                CreateFolder(principalID, rootFolder.ID, 50, "Inbox");
+            if (!Array.Exists(sysFolders, delegate (XInventoryFolder f) { if (f.type == 57) return true; return false; }))
+                CreateFolder(principalID, rootFolder.ID, 57, "Materials");
 
             return result;
         }
