--- /tmp/core_clean/OpenSim/Region/ClientStack/Linden/UDP/LLClientView.cs	2026-01-05 18:07:33.090901242 +0000
+++ /tmp/core_work/OpenSim/Region/ClientStack/Linden/UDP/LLClientView.cs	2026-01-05 18:07:45.302121467 +0000
@@ -8804,6 +8804,9 @@
                 Scene = c.Scene,
                 Sender = c
             };
+
+            EncounterLogger.Log("Ranger", "TERRITORY", "SIGNAL", args.Message);
+
             c.OnChatFromClient?.Invoke(c, args);
         }
