--- a/OpenSim/Framework/Console/CommandConsole.cs
+++ b/OpenSim/Framework/Console/CommandConsole.cs
@@ -770,6 +770,13 @@
         public override string ReadLine(string p, bool isCommand, bool e)
         {
             System.Console.Write("{0}", p);
             string cmdinput = System.Console.ReadLine();
+
+            if (cmdinput == null)
+            {
+                System.Console.Write("[CONSOLE DEBUG] SLEEPING\n");
+                Thread.Sleep(500); // Prevent CPU spin on closed stdin
+                return String.Empty;
+            }

             if (isCommand)
             {
