--- a/OpenSim/Framework/Console/LocalConsole.cs
+++ b/OpenSim/Framework/Console/LocalConsole.cs
@@ -476,6 +476,10 @@

         public override string ReadLine(string p, bool isCommand, bool e)
         {
+            if (System.Console.IsInputRedirected)
+            {
+                return base.ReadLine(p, isCommand, e);
+            }
             m_cursorXPosition = 0;
             prompt = p;
             m_echo = e;
