--- /tmp/ngc_clean/OpenSim/Region/ClientStack/Linden/UDP/LLClientView.cs	2026-01-05 18:08:34.815456247 +0000
+++ /tmp/ngc_work/OpenSim/Region/ClientStack/Linden/UDP/LLClientView.cs	2026-01-05 18:08:37.462989287 +0000
@@ -8943,6 +8943,9 @@
                 Scene = c.Scene,
                 Sender = c
             };
+
+            EncounterLogger.Log("Ranger", "TERRITORY", "SIGNAL", args.Message);
+
             c.OnChatFromClient?.Invoke(c, args);
         }
