diff --git a/crates/messages/src/udp/chat/chat_from_viewer.rs b/crates/messages/src/udp/chat/chat_from_viewer.rs
index e1d02af..57550ae 100644
--- a/crates/messages/src/udp/chat/chat_from_viewer.rs
+++ b/crates/messages/src/udp/chat/chat_from_viewer.rs
@@ -21,7 +21,7 @@ impl Packet {
             header: Header {
                 id: 80,
                 frequency: PacketFrequency::Low,
-                reliable: false,
+                reliable: true,
                 zerocoded: false,
                 ..Default::default()
             },
@@ -64,6 +64,11 @@ impl PacketData for ChatFromViewer {
         let mut message_bytes = vec![0u8; message_length];
         cursor.read_exact(&mut message_bytes)?;
 
+        // Trim null terminator if present
+        if let Some(&0) = message_bytes.last() {
+            message_bytes.pop();
+        }
+
         let message = String::from_utf8(message_bytes)?;
 
         let message_type_byte = cursor.read_u8()?;
@@ -94,8 +99,6 @@ impl PacketData for ChatFromViewer {
         bytes.push(self.message_type.to_bytes());
         bytes.extend_from_slice(&self.channel.to_le_bytes());
 
-        bytes.push(0);
-
         bytes
     }
 }
